gateway-ha/src/main/java/io/trino/gateway/ha/router/QueryCountBasedRouter.java

```diff
@@ -46,7 +47,7 @@ static class LocalStats
     {
         private int runningQueryCount;
         private int queuedQueryCount;
-        private boolean healthy;
+        private TrinoHealthStateType healthy;
```

Comment:
question: what are your thoughts on having this be `heathState` instead of `healthy`?

as well as respective places where `healthy` is used